app/Services/GitHub/ConsolidateOrganizationsService.php

```
@@ -0,0 +1,111 @@
+            }
+        }
+
+        return $stats;
+    }
+
+    /**
+     * Transfer a single repository to the target organization.
+     *
+     * @return string 'transferred', 'failed', or 'skipped'
+     */
+    private function transferRepository(Repository $repository, string $targetOrganizationLogin): string
```

PHPMD (2.15.0)

Avoid excessively long variable names like $targetOrganizationLogin. Keep variable name length under 20. (undefined)

(LongVariable)